### DISCUSSION TITLE
License key randomly not auto-renewing / reverting to an old key?

### DISCUSSION THREAD

**Discussion by unicornsoftwareinc** [ant-media/Ant-Media-Server#6565] [Q&A]
Hi, we're having an issue in the past few weeks where for some reason, our Ant Media license seems to be having issues auto-renewing or even reverting to a different (older?) license key which effectively blocks us from using our servers. 

This has happened twice so far which has blocked our customers from using our tool and made some of customers, understandably, very angry.

We've been using Ant Media for years at this point and have never encountered this so we're not sure what's going on. Has anyone else encountered this?

Is there some built-in way to get notifications when the license changes so that we can immediately correct this issue if it happens? Is there an API on the server we can hit to detect and even set the license key?

**Comment by muratugureminoglu** [comment]
Hi @unicornsoftwareinc 

I’m sorry to hear about the issue you’ve encountered. It’s true that some accounts have faced a similar problem, and our developer team is actively working on it.

I’ve checked and tested your license key, and I can confirm that there is no blocking issue. Instead, you are seeing an "invalid license" error, which only appears on the dashboard when you log in. Your license has not been blocked, and this issue does not cause any service interruption.Additionally, I will create a workaround solution to ensure that you do not encounter this issue again.

I will review this matter with the developer team and keep you updated through the ticket you’ve opened.

Thank you for your understanding.
Regards.

**Comment by unicornsoftwareinc** [comment]
Hi Murat, thanks for your response, I am relieved to hear that it wasn't just us that experienced this and that the team is actively working on a solution. 

However I just want to make clear, for us, there was a service disruption both times this happened. It was not just a warning message in the dashboard. 

Myself and our customers were not able to publish streams / go live until I manually replaced the license key with the newest one emailed to us. So there was definitely a service disruption unfortunately.

Any updates on whatever your team does to prevent this from happening in the future would be much appreciated.

**Comment by muratugureminoglu** [comment]
Hi @unicornsoftwareinc 

A workaround solution has been implemented for this situation.  Please check it.

**Comment by unicornsoftwareinc** [comment]
Hi Murat, great to hear, where can I see this workaround solution?

**Comment by muratugureminoglu** [comment]
Hi,

It has been fixed in backend side. You can see the results in settings/license page. Please keep using old license key.
